rest/rest-client/src/test/java/functionaltests/SchedulerClientTest.java

```diff
@@ -543,7 +543,7 @@ public void testReSubmitJobMergeInfo() throws Exception {
         Map<String, String> info1 = new HashMap<>();
         info1.put("newVar", "newValue");
 
-        JobId jobId1 = client.reSubmit(jobId, Collections.emptyMap(), info1);
+        JobId jobId1 = client.reSubmit(jobId, Collections.emptyMap(), info1, null);
 
         String jobContent1 = client.getJobContent(jobId1);
 
```

Comment:
Is it possible to add a functional test with a not null sessionid ? or it is rather a system test to be added ?

Reply:
Since sessionId is only used in the catalog rest request, the functional test with not-null sessionId will depend on the service of catalog. I think system test would be more proper. I'll add it later. Thank you.